dblookupext/errors.go

```diff
@@ -4,8 +4,18 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"strings"
 )
 
+// ErrNotFoundInStorage signals that an item was not found in storage
+var ErrNotFoundInStorage = errors.New("not found in storage")
+
+// Currently, "item not found" storage errors are untyped (thus not distinguishable from others). E.g. see "pruningStorer.go".
+// As a workaround, we test the error message for a match.
+func isNotFoundInStorageErr(err error) bool {
```

Comment:
might add an `if err == nil { return false }` to avoid any lead to null pointers

Reply:
Fixed.